kalite/distributed/management/commands/kaserve.py

```diff
@@ -149,7 +149,13 @@ def handle(self, *args, **options):
             call_command("runserver", "%s:%s" % (options["host"], options["port"]))
         else:
             del options["production"]
+            addresses = get_ip_addresses(include_loopback=False)
+            loopbacks = set(get_ip_addresses(include_loopback=True)) - set(addresses)
```

Comment:
So we only want the loopback address? I think it's always `127.0.0.1`. If not then they're using a strange computer.